com.unity.netcode.gameobjects/Components/NetworkAnimator.cs

```diff
@@ -317,77 +326,69 @@ internal struct AnimationState : INetworkSerializable
             internal float NormalizedTime;
             internal int Layer;
             internal float Weight;
+            internal float Duration;
 
             // For synchronizing transitions
             internal bool Transition;
+            internal bool CrossFade;
+
+            // Flags for bool states
+            private const byte k_IsTransition = 0x01;
+            private const byte k_IsCrossFade = 0x02;
+
+            // Used to serialize the bool states
+            private byte m_StateFlags;
+
             // The StateHash is where the transition starts
             // and the DestinationStateHash is the destination state
             internal int DestinationStateHash;
 
             public void NetworkSerialize<T>(BufferSerializer<T> serializer) where T : IReaderWriter
             {
-                if (serializer.IsWriter)
+                var isWriter = serializer.IsWriter;
+                if (isWriter)
```

Comment:
Just curious but why make this change?

Reply:
Good catch!  I originally was going to be checking that at other locations but ended up serializing in a way that did not need that.  There is no reason for that adjustment and will be committing an update that removes it.  👍